### PR TITLE
Support for more symbols ! ? + = × ÷

### PR DESCRIPTION
**Pull request by 84ds84d8s**
somehow i cant get "counter = 0
for emojis in "⿠ ⿡ ⿢ ⿣ ⿤ ⿥ ⿦ ⿧ ⿨ ⿩".split():
    convert_dict[counter] = emojis
    counter += 1" to work.

### CODE REVIEW

File: text2emoji.py [hunk 1]
```diff
@@ -1,20 +1,46 @@
...
+    for character in input():
     # use the dict.get() method since a lot of characters not included in the convert_dict
-    print(convert_dict.get(character.upper(), character), end=" ")
-input("Press any key to exit . . .")
+        print(convert_dict.get(character.upper(), character), end=" ")
+    x = input("Press 1 to continue, any other key to exit: ")
+    if x == "1":
+        main()
+    else:
+        exit()
```
Comment (eric15342335) [inline text2emoji.py]: I think this `exit()` is redundant. You can just let the function complete itself.
i.e. change
```python
if x == "1":
    main()
else:
    exit()
```
to
```python
if x == "1":
    main()
```

File: text2emoji.py [hunk 2]
```diff
@@ -1,20 +1,46 @@
+def main():
 # the dictionary that stores the character and their emoji counterpart
-convert_dict = {}
+    convert_dict = {}
 
 # ASCII 'A' is 65
```
Comment (eric15342335) [inline text2emoji.py]: Indent the comments too.

**Comment by eric15342335** [comment]
You should summarize what you have done in the commit message. E.g. "Support for more symbols ! ? + = × ÷" instead of "Update text2emoji.py".
This is required for our group project as well.

File: text2emoji.py [hunk 3]
```diff
@@ -1,20 +1,46 @@
...
+        counter += 1
 
 # numerical emojis
-counter = 0
-for emojis in "⿠ ⿡ ⿢ ⿣ ⿤ ⿥ ⿦ ⿧ ⿨ ⿩".split():
-    convert_dict[counter] = emojis
-    counter += 1
+    counter = 48
+    for emojis in "⿠ ⿡ ⿢ ⿣ ⿤ ⿥ ⿦ ⿧ ⿨ ⿩".split():
+        convert_dict[chr(counter)] = emojis
```
Comment (eric15342335) [inline text2emoji.py:15]:
```
somehow i cant get "counter = 0
for emojis in "⿠ ⿡ ⿢ ⿣ ⿤ ⿥ ⿦ ⿧ ⿨ ⿩".split():
convert_dict[counter] = emojis
counter += 1" to work.
```
This is why it does not work : you changed `convert_dict[counter] = emojis` to `convert_dict[chr(counter)] = emojis`. Which the ASCII characters from `chr(0)` to `chr(9)` are not what we want.

You can see it here: https://www.ascii-code.com/

File: text2emoji.py [hunk 4]
```diff
@@ -1,20 +1,46 @@
...
+        counter += 88 #158+88 = 246 is the ascii for division
+        
+    for character in input():
     # use the dict.get() method since a lot of characters not included in the convert_dict
-    print(convert_dict.get(character.upper(), character), end=" ")
-input("Press any key to exit . . .")
+        print(convert_dict.get(character.upper(), character), end=" ")
+    x = input("Press 1 to continue, any other key to exit: ")
+    if x == "1":
```
Comment (eric15342335) [inline text2emoji.py]: You don't need to define a new variable `x`.
```python
x = input("Press 1 to continue, any other key to exit: ")
if x == "1":
```
is same as
```python
if input("Press 1 to continue, any other key to exit: ") == "1":
```
But it's fine to leave it as is. Just let you know.

**Review by eric15342335 — CHANGES_REQUESTED** [review]

File: text2emoji.py (hunk 2 above)
Comment (eric15342335) [reply text2emoji.py]: Remember to indent other comments as well:
```
text2emoji.py:17:0: C0303: Trailing whitespace (trailing-whitespace)
text2emoji.py:22:0: C0303: Trailing whitespace (trailing-whitespace)
text2emoji.py:23:16: C0303: Trailing whitespace (trailing-whitespace)
text2emoji.py:27:0: C0303: Trailing whitespace (trailing-whitespace)
text2emoji.py:37:0: C0303: Trailing whitespace (trailing-whitespace)
```

**Comment by eric15342335** [comment]
> somehow i cant get "counter = 0 for emojis in "⿠ ⿡ ⿢ ⿣ ⿤ ⿥ ⿦ ⿧ ⿨ ⿩".split(): convert_dict[counter] = emojis counter += 1" to work.

Good fix!
The actual problem is that I forget to use the string form of the counters, the correct way do it should be:
```python
counter = 0
for emojis in "⿠ ⿡ ⿢ ⿣ ⿤ ⿥ ⿦ ⿧ ⿨ ⿩".split():
convert_dict[str(counter)] = emojis     # <--- notice the key is str(counter) instead of counter
counter += 1
```
But your fix also works flawlessly!

**Review by eric15342335 — APPROVED** [review]

**Comment by eric15342335** [comment]
@84ds84d8s I have pushed a updated version.

**Comment by 84ds84d8s** [comment]
how should i approve it

**Comment by eric15342335** [comment]
Go to "file changed" and find if there was a "Review changes" button

**Comment by 84ds84d8s** [comment]
let me do that later, i am busy

**Comment by eric15342335** [comment]
ok

**Comment by eric15342335** [comment]
Hmm, [pull request authors cannot approve their own pull requests](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews#about-pull-request-reviews). Perhaps we could go ahead and merge this to keep things moving.

**Pull request merged**